tool/sdg/src/main/java/org/openscience/cdk/layout/RingPlacer.java

```diff
@@ -523,18 +498,43 @@ public void placeFusedRing(IRing ring,
             atomsToDraw.addElement(currentAtom);
         }
         addAngle = addAngle * direction;
-        try {
-            logger.debug("placeFusedRing->startAngle: " + Math.toDegrees(startAngle));
-            logger.debug("placeFusedRing->addAngle: " + Math.toDegrees(addAngle));
-            logger.debug("placeFusedRing->startAtom is: " + (molecule.getAtomNumber(startAtom) + 1));
-            logger.debug("AtomsToDraw: " + AtomPlacer.listNumbers(molecule, atomsToDraw));
-        } catch (Exception exc) {
-            logger.debug("Caught an exception while logging in RingPlacer");
-        }
+
+        logger.debug("placeFusedRing->startAngle: " + Math.toDegrees(startAngle));
+        logger.debug("placeFusedRing->addAngle: " + Math.toDegrees(addAngle));
+        logger.debug("placeFusedRing->startAtom is: " + (molecule.getAtomNumber(startAtom) + 1));
+        logger.debug("AtomsToDraw: " + AtomPlacer.listNumbers(molecule, atomsToDraw));
+
         atomPlacer.populatePolygonCorners(atomsToDraw, ringCenter, startAngle, addAngle, radius);
     }
 
     /**
+     * Completes the layout of a partiallyed laid out ring.
+     *
+     * @param rset ring set
+     * @param ring the ring to complete
+     * @param bondLength the bond length
```

Comment:
Yep it will look dodgey, but the API is low enough with the atom/bond locking that the invokee must handle it. It's the most flexible and also means we don't have worry about cases like this - if someone actually wanted to do that it's possible.